calcom/cal.com
# Admin Wizard Choose License

## What does this PR do?

Implements 2 more steps in the Admin wizard for self-hosters to choose a license.

Closes #5998 #6014

**Environment**: Staging(main branch) / Production

Loom Video: UPDATED: https://loom.com/share/0d58e2fa5683478ebb52954b92eb6ee9

## Type of change


## How should this be tested?

Go to `/auth/setup` in a self-hosted Cal.com and see the wizard come up or If you don't have any user created in the database it will come up automatically when visiting the app. You can see two more steps or just one depending on the license chosen after setting up the first user (admin).
